x-pack/plugins/security_solution/server/endpoint/routes/resolver/queries/events.ts

```diff
@@ -4,86 +4,59 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { SearchResponse } from 'elasticsearch';
+import { IScopedClusterClient } from 'kibana/server';
```

Comment:
This is really a new file for the new `/resolver/events` route. It does not use the base class like the other queries that are tied to a particular entity_id.